### DISCUSSION TITLE
ESP32 C3

### DISCUSSION THREAD

**Discussion by wyc23** [HomeSpan/HomeSpan#281] [Q&A]
will it work for esp32 c3?

**Comment by wyc23** [comment]
i fail to compile for c3, here is the error info:


"ESP32 Dev Module, Disabled, Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS), 240MHz (WiFi/BT), QIO, 80MHz, 4MB (32Mb), 921600, Core 1, Core 1, None"

In file included from C:\Users\wyc\Documents\Arduino\libraries\HomeSpan\src/HomeSpan.h:41,

                 from C:\Users\wyc\Documents\Arduino\libraries\HomeSpan\examples\05-WorkingLED\05-WorkingLED.ino:39:

C:\Users\wyc\Documents\Arduino\libraries\HomeSpan\src/Settings.h:30:10: fatal error: core_version.h: No such file or directory

 #include <core_version.h>

          ^~~~~~~~~~~~~~~~

compilation terminated.

exit status 1

**Comment by wyc23** [comment]
it seems like the <core_version.h> can't be found

**Reply by HomeSpan** [reply]
This can occur if you have not properly installed the Arduino-ESP32 board manager.  See issue #202

**Reply by wyc23** [reply]
Thank you very much, I’ve installed the proper esp32 board manager and solved the issue 🥳
